src/types/identity.ts

```diff
@@ -28,6 +28,19 @@ export interface PublicUserInfo {
   version: PrivateUserVersion;
 }
 
+export interface DerivedUserInfo {
```

Comment:
DH: nit: Change this to DerivedPrivateUserInfo since this contains key material.